packages/core/admin/admin/src/pages/ProfilePage/index.js

```diff
@@ -145,126 +162,203 @@ const ProfilePage = () => {
-                  <Box
-                    background="neutral0"
-                    hasRadius
-                    shadow="filterShadow"
-                    paddingTop={6}
-                    paddingBottom={6}
-                    paddingLeft={7}
-                    paddingRight={7}
-                  >
-                    <Stack size={4}>
-                      <H3 as="h2">
-                        {formatMessage({
-                          id: 'Settings.profile.form.section.experience.title',
-                          defaultMessage: 'Experience',
-                        })}
-                      </H3>
-                      <Select
-                        label={formatMessage({
-                          id: 'Settings.profile.form.section.experience.interfaceLanguage',
-                          defaultMessage: 'Interface language',
-                        })}
-                        placeholder={formatMessage({
-                          id: 'components.Select.placeholder',
-                          defaultMessage: 'Select',
-                        })}
-                        hint={formatMessage({
-                          id: 'Settings.profile.form.section.experience.interfaceLanguage.hint',
-                          defaultMessage:
-                            'This will only display your own interface in the chosen language.',
-                        })}
-                        onClear={() =>
-                          handleChange({ target: { name: 'preferedLanguage', value: null } })}
-                        clearLabel={formatMessage({
-                          id: 'Settings.profile.form.section.experience.clear.select',
-                          defaultMessage: 'Clear the interface language selected',
-                        })}
-                        value={values.preferedLanguage}
-                        onChange={e =>
-                          handleChange({ target: { name: 'preferedLanguage', value: e } })}
-                      >
-                        {Object.keys(localeNames).map(language => {
-                          const langName = localeNames[language];
+              <Box paddingBottom={10}>
+                <ContentLayout>
+                  <Stack size={6}>
+                    <Box
+                      background="neutral0"
+                      hasRadius
+                      shadow="filterShadow"
+                      paddingTop={6}
+                      paddingBottom={6}
+                      paddingLeft={7}
+                      paddingRight={7}
+                    >
+                      <Stack size={4}>
+                        <H3 as="h2">
+                          {formatMessage({
+                            id: 'Settings.profile.form.section.profile.title',
+                            defaultMessage: 'Profile',
+                          })}
+                        </H3>
+                        <Grid gap={5}>
+                          {layout.map(input => {
```

Comment:
It seems a bit too complex to use a layout here, maybe you can "hardcode" the inputs?